docs/modeling/connectors/ootb/content.md

```diff
@@ -0,0 +1,367 @@
+| `search_orderBy` | *Optional.* Orders the list of properties returned by the `searchQuery`. | Array | |
+| `search_fields` | *Optional.* Set the list of properties to be returned about the node in the response of the `searchQuery` | Array | id, name |
+
+`*` One of these parameters is required to identify the content to operate on. 
+
+### Common output parameters 
+
+| Parameter | Description | Type | Example |
+| --------- | ----------- | ---- | ------- | 
+| `nodes` | *Optional.* A list of nodes operated on by the action. | Array | | 
+| `responses` | *Required.* The response from the action. For example the metadata of a node. | JSON | |
+| `content.error` | *Optional.* Any errors returned by the connector in performing the operation. | String | | 
+
+
+### Create a file
+The `CREATE_NODE` action is used to upload files to the Alfresco Content Services repository. 
+
+| Parameter | Description | Type | Example |
+| --------- | ----------- | ---- | ------- |
+| `name` | *Required.* A name for the file. | String | January Orders | 
+| `autorename` | *Optional.* If set to `true` then a suffix will be added to the file name if a file with that name already exists.| Boolean | True | 
+| `nodeType` | *Optional.* The node type for the new content. The default value is `cm:content` | String | cm:content | 
+| `include` | *Optional.* Additional information to return about the node. Values include: `properties`, `aspectNames`, `path`, `isLink`, `allowableOperations`, `association`, `isLocked`, `permissions`. | Array | properties, permissions | 
+| `fields` | *Optional.* Set the list of properties to be returned about the node. | Array | id, name |
+
+### Create a folder
+The `CREATE_FOLDER` action is used to create new folders in the Alfresco Content Services repository.
+
+| Parameter | Description | Type | Example |
+| --------- | ----------- | ---- | ------- |
+| `folderName` | *Required.* A name for the folder. | String | 2020-shipments | 
+| `autorename` | *Optional.* If set to `true` then a suffix will be added to the folder name if a folder with that name already exists.| Boolean | True | 
+| `nodeType` | *Optional.* The node type for the new content. The default value is `cm:file` | String | cm:file | 
+| `include` | *Optional.* Additional information to return about the node. Values include: `properties`, `aspectNames`, `path`, `isLink`, `allowableOperations`, `association`, `isLocked`, `permissions`. | Array | properties, permissions | 
+| `fields` | *Optional.* Set the list of properties to be returned about the node. | Array | id, name |
+
+### Update content
+The `UPDATE_CONTENT` action is used to upload new versions of files to the Alfresco Content Services repository.
+
+| Parameter | Description | Type | Example |
+| --------- | ----------- | ---- | ------- |
+| `name` | *Optional.* A name for the file. | String | January Orders | 
```

Comment:
This should include the file extension.

Reply:
Is this just for `CREATE_NODE` or for `UPDATE_CONTENT` as well?

Reply:
It is for UPDATE_CONTENT

Reply:
👍